value/5003/meta.json

```diff
@@ -0,0 +1,13 @@
+{
+    "value": "Honor",
+    "military": [
+        {
+            "branch": "Army",
+            "definition": "Live up to Army values [...] Honor is a matter of carrying out, acting, and living the values of respect, duty, loyalty, selfless service, integrity and personal courage in everything you do."
+        },
+        {
+            "branch": "Marines / Navy",
+            "definition": "The quality of maturity, dedication, trust, and dependability that commits Marines to act responsibly; to be accountable for actions; to fulfill obligations; and to hold others accountable for their actions."
+        }
+    ]
```

Comment:
This value submodule has `military` property to list the branches and definitions whereas the other values submodule has `inspiration`. I think they should both be the same shape. Either use `military` or `inspirations` but not both

Reply:
thanks for the catch! I think some of my vscode changes didn't save :(